hr_holidays_compute_days/models/hr_holidays.py

```diff
@@ -5,7 +5,6 @@
 from openerp import models, fields, api, _
 from openerp.exceptions import ValidationError
 from dateutil.relativedelta import relativedelta
-import math
 
 
 class HrHolidays(models.Model):
```

Comment:
@yelizariev Can you please add an onchange method for holiday_status_id, the default view is the calendar view, when you select dates, since the holiday_status_id is not selected and not even in the context, the method is not executed, after you select dates it will open the form view of the holiday where you need to select the holiday_status_id, but the method of computing days is not triggered at tjis change...so if you will not change dates or employee, the resulted number of days will be wrong.

And please add test for the method.